tutorials/tutorial-howto-monitor-django-prometheus.md

```diff
@@ -0,0 +1,230 @@
+# TYPE python_gc_objects_collected_total counter
+python_gc_objects_collected_total{generation="0"} 11716.0
+python_gc_objects_collected_total{generation="1"} 1699.0
+python_gc_objects_collected_total{generation="2"} 616.0
+# HELP python_gc_objects_uncollectable_total Uncollectable object found during GC
+# TYPE python_gc_objects_uncollectable_total counter
+python_gc_objects_uncollectable_total{generation="0"} 0.0
+python_gc_objects_uncollectable_total{generation="1"} 0.0
+python_gc_objects_uncollectable_total{generation="2"} 0.0
+# HELP python_gc_collections_total Number of times this generation was collected
+# TYPE python_gc_collections_total counter
+python_gc_collections_total{generation="0"} 7020.0
+python_gc_collections_total{generation="1"} 638.0
+python_gc_collections_total{generation="2"} 34.0
+# HELP python_info Python platform information
+# TYPE python_info gauge
+python_info{implementation="CPython",major="3",minor="8",patchlevel="0",version="3.8.0"} 1.0
+...
+```
+
+### Step 2 - Point Prometheus to your Django application metrics endpoint [](step2)
+
+*(Note: This section assumes that you have a locally running Prometheus
+instance.)*
+
+#### Update `prometheus.yml`. Under `scrape_configs:`, add:
+
+```yaml
+- job_name: django
+  scrape_interval: 10s
+  static_configs:
+  - targets:
+    - localhost:8000
+```
+
+>:TIP: Replace the job_name “django” with whatever you’d like as a prefix for
+your Django application metrics in Prometheus. For example, “webapp”, etc.)
+
+#### Restart Prometheus
+```bash
+./prometheus --config.file=prometheus.yml
+```
+
+#### Verify that Prometheus is scraping metrics from yourDjango application:
+Visit the [Prometheus Expression Browser][localhost-prom-browser] from your
+web browser. 
+
+For example, you can visit the below page, which graphs the total number of
+http requests your Django application received in the last hour:
+
+[http://localhost:9090/graph?g0.range_input=1h&g0.stacked=1&g0.expr=django_http_requests_total_by_method_total&g0.tab=0][localhost-prom-example]
+
+It should look something like this:
+
+<img class="main-content__illustration" src="https://assets.iobeam.com/images/docs/screenshots-for-tutorial-django-prometheus/prom_expression_browser.png" alt="Prometheus graph of Total HTTP Requests in the last hour"/>
+
+If you’d like to do more testing, visit your Django application several
+more times and reload the Prometheus Expression Browser to confirm that it
+is working. Also feel free to explore the other Django metrics that
+Prometheus is collecting.
+
+### Step 3 - Instrument additional aspects of your application (optional) [](#step3)
+[Django-prometheus][get-django-prometheus] is quite powerful, and allows
+you to easily instrument additional aspects of your application, including:
+
+* Your databases
+* Your models (e.g., monitor the creation/deletion/update rate for your models)
+* Your caches
+* You can even create your own custom metrics in your code.
```

Comment:
This entry doesn't fit the tenses of the first three 
> allows you to easily instrument additional aspects of your application, including: You can even create your own custom metrics in your code.

Consider taking it out of the list and placing it in a line directly below

Reply:
Reworded.